PhotoGether/PresentationLayer/EditPhotoRoomFeature/EditPhotoRoomFeature/Source/StickerBottomSheetViewController.swift

```diff
@@ -0,0 +1,127 @@
+import Combine
+import UIKit
+
+import BaseFeature
+import DesignSystem
+import PhotoGetherDomainInterface
+
+protocol StickerBottomSheetViewControllerDelegate {
+    func stickerBottomSheetViewController(_ viewController: StickerBottomSheetViewController, didTap emoji: EmojiEntity)
+}
+
+final class StickerBottomSheetViewController: UIViewController, ViewControllerConfigure {
+    private let collectionView: StickerCollectionView
+    private let viewModel: StickerBottomSheetViewModel
+    
+    var delegate: StickerBottomSheetViewControllerDelegate?
+    
+    private let input = PassthroughSubject<StickerBottomSheetViewModel.Input, Never>()
+    private var cancellables = Set<AnyCancellable>()
+    
+    init(viewModel: StickerBottomSheetViewModel) {
+        let layout = UICollectionViewFlowLayout()
+        self.collectionView = StickerCollectionView(collectionViewLayout: layout)
+        self.viewModel = viewModel
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
```

Comment:
```suggestion
    required init?(coder: NSCoder) { 
        fatalError("init(coder:) has not been implemented") 
    }
```
다른 곳도 전부 내려쓰고 있어서 통일하면 어떨까요?

Reply:
간지 뒤지잖아요 😎